src/features/apiDelegation/received/OverviewPage/OverviewPage.tsx

```diff
@@ -15,7 +15,7 @@ export const OverviewPage = () => {
     <div>
       <PageContainer>
         <Page
-          color={PageColor.Light}
+          color={PageColor.Dark}
```

Comment:
You would probably need to change more than one place to get the dark header on all the pages, but I've done it in my PR as well so it doesn't really matter

Reply:
Oh that's right. I don't think I will make it to finish review on your PR today so I'll change the rest of the colors